feat(trace-eap-waterfall): Adding highlight section for eap span details

Abdkhan14 opened the pull request:
<img width="1288" alt="Screenshot 2025-04-22 at 12 01 29 AM" src="https://github.com/user-attachments/assets/bf8ba3a0-6aaa-43ee-9c89-ab19be213b2d" />

Commit:
feat(trace-eap-waterfall): Adding highlight section for eap span details

Commit:
feat(trace-eap-waterfall): Fixing duration comparison text

Commit:
feat(trace-eap-waterfall): Fixing tests

Commit:
feat(trace-eap-waterfall): Fixing eslint errors

edwardgou-sentry commented on static/app/views/insights/common/queries/useSpansQuery.tsx:
```diff
@@ -50,58 +40,104 @@ export function useSpansQuery<T = any[]>({
+    enabled: (props.enabled || props.enabled === undefined) && pageFiltersReady,
+    referrer,
+    trackResponseAnalytics,
+    withPageFilters: true,
```
nit: Instead of building out separate functions for with and without page filters, could we just add `withPageFilters` as a prop to `SpansQueryProps` (with a default value of `true`)? That way, any time `useSpansQuery` is called, the user can set `withPageFilters` to true or false, and the setting can propagate to `useWrappedDiscoverTimeseriesQuery` or `useWrappedDiscoverQuery`.

Abdkhan14 replied:
Yep probably a bit of a stretch to add separate functions here I'll follow up with a PR

edwardgou-sentry reviewed (approved):
lgtm, left some suggestions

Commit:
feat(trace-eap-watefall): Resolving conflicts

Abdkhan14 committed:
feat(trace-eap-waterfall): Adding opsbreakdown to span description (#90072)

<img width="1229" alt="Screenshot 2025-04-22 at 12 33 37 PM"
src="https://github.com/user-attachments/assets/fce9344c-7c1b-4586-83a9-9cad8f6a4d79"
/>

---------

Co-authored-by: Abdullah Khan <abdullahkhan@PG9Y57YDXQ.local>
